impl/pom.xml

```diff
@@ -629,17 +629,16 @@
                                 </Export-Package>
                                 <Import-Package>
                                     !org.apache.myfaces.*,
-                                    javax.annotation,
+                                    jakarta.annotation,
                                     javax.crypto,
                                     javax.crypto.spec,
-                                    javax.ejb;resolution:=optional,
-                                    javax.el;version="[1.0.0, 3.1.0)",
+                                    jakarta.ejb;resolution:=optional,
+                                    jakarta.el;version="[5.0.0, 6.0.0)",
                                     javax.naming,
-                                    javax.persistence;version="[1.0.0, 2.1)";resolution:=optional,
-                                    javax.portlet;version="[1.0.0, 2.1)";resolution:=optional,
```

Comment:
@pnicolucci @tandraschko  Any objections to removing javax.portlet as there's no EE9 and EE10 equivalent?  

However, the faces 4.0 spec still mentions portlet.    

1) Remove portlet import
2) change to jakarta.portlet 
3) leave as javax.portlet (no change) 


With option two, there's one portlet search hit here: https://github.com/apache/myfaces/blob/e9fe59f96410f31a7f5c0fbd6838c1a22683a691/impl/src/main/java/org/apache/myfaces/util/ExternalContextUtils.java#L41

Reply:
Mojarra doesn't have any portlet imports either. I think option 1 is best. I created https://issues.apache.org/jira/browse/MYFACES-4472 as a follow up

Reply:
My vote would be to keep it as it until MYFACES-4472 is investigated and completed.

Reply:
I've added it back in.